docs/uml/main/OutputDatamodelConcept.puml

```diff
@@ -82,10 +82,14 @@ package models {
             class PvResult
             SystemParticipantResult  <|-- PvResult
 
-            class ChpResult
+            class ChpResult{
+                + qDot: ComparableQuantity<Power>
+            }
             SystemParticipantResult <|-- ChpResult
```

Comment:
The UML documentation is not correct. `ChpResult` and `HpResult` inherit from `SystemParticipantWithHeatResult`. That one is missing completely. Additionally, `qDot` then is part of `SystemParticipantWithHeatResult`.

Reply:
Sure, I will make the changes.